database/migrations/20190822101845-create-mentoring-types.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars */
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Mentoring_types', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
```

Comment:
I'm being really really picky, but you could put table key/values in the same order so it's consistent and easy to read/review.